### PR TITLE
Enable-MySQL-4-byte

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -246,7 +246,7 @@ then
     exec_occ -V
 
     if [ "$(exec_occ config:system:get mysql.utf8mb4)" != "true" ]; then
-        exec_occ db:convert-mysql-charset
+        exec_occ maintenance:repair
```
Comment (JimboJoe) [inline scripts/upgrade:249]: Shouldn't you set also the `mysql.utf8mb4` attribute to true...?

Comment (ericgaspar) [reply scripts/upgrade:256]: True...  I guess we also need to alter the database character set... (I am not able to test this on my setup for now...)

**Comment by ericgaspar** [comment]
!testme

**Review by JimboJoe — APPROVED** [review]
Tested on my production server and that worked well (it takes a lot of time) 👍

**Comment by JimboJoe** [comment]
From the (official doc)[https://docs.nextcloud.com/server/22/admin_manual/configuration_database/mysql_4byte_support.html], aren't we missing this part:
>Also make sure your backup strategy still work. If you use mysqldump make sure to add the --default-character-set=utf8mb4 option. Otherwise your backups are broken and restoring them will result in ? instead of the emojis, making files inaccessible.

I'm a bit puzzled because wallabag2 already manages utf8mb4 charset, but there's nothing specific in the package... 🤔

**Pull request merged**